### PR TITLE
Bug 1370271 - Remove support for ios9. Update deployment target to 10.3.

### PR DESCRIPTION
**Pull request by farhanpatel**
1vgvr7

### CODE REVIEW

**Comment by farhanpatel** [comment]
### SwiftLint found issues

#### Warnings

File | Line | Reason |
| --- | ----- | ----- |
TabTrayController.swift | 471 | Colons should be next to the identifier when specifying a type and next to the key in dictionary literals. 
TabTrayController.swift | 1142 | Colons should be next to the identifier when specifying a type and next to the key in dictionary literals. 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by farhanpatel** [comment]
@jhugman I removed some obvious code paths related to local notifications used in ios9, if there are more could you please update this PR.

File: Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift [hunk 1]
```diff
@@ -88,30 +88,8 @@ extension BrowserViewController: WKNavigationDelegate {
...
-                    let alert = UIAlertController(title: formattedPhoneNumber, message: nil, preferredStyle: UIAlertControllerStyle.alert)
-                    alert.addAction(UIAlertAction(title: NSLocalizedString("Cancel", comment:"Label for Cancel button"), style: UIAlertActionStyle.cancel, handler: nil))
-                    alert.addAction(UIAlertAction(title: NSLocalizedString("Call", comment:"Alert Call Button"), style: UIAlertActionStyle.default, handler: { (action: UIAlertAction!) in
-                        UIApplication.shared.openURL(url)
-                    }))
-                    present(alert, animated: true, completion: nil)
-                }
-            }
-
+            UIApplication.shared.openURL(url)
```
Comment (jhugman) [inline Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift]: Woah.

File: Client/Frontend/Browser/TabTrayController.swift [hunk 2]
```diff
@@ -726,18 +726,6 @@ extension TabTrayController: TabManagerDelegate {
                 guard finished && self.privateTabsAreEmpty() else { return }
                 self.emptyPrivateTabsView.isHidden = false
             })
-
-            // Workaround: On iOS 8.* devices, cells don't get reloaded during the deletion but after the
-            // animation has finished which causes cells that animate from above to suddenly 'appear'. This
-            // is fixed on iOS 9 but for iOS 8 we force a reload on non-visible cells during the animation.
-            if floor(NSFoundationVersionNumber) <= NSFoundationVersionNumber_iOS_8_3 {
```
Comment (jhugman) [inline Client/Frontend/Browser/TabTrayController.swift]: !!

File: Client/Helpers/FxALoginHelper.swift [hunk 3]
```diff
@@ -178,49 +178,13 @@ class FxALoginHelper {
...
-
-        let sentTabsCategory = UIMutableUserNotificationCategory()
-        sentTabsCategory.identifier = TabSendCategory
-        sentTabsCategory.setActions([readingListAction, bookmarkAction, viewAction], for: .default)
-
-        sentTabsCategory.setActions([bookmarkAction, viewAction], for: .minimal)
-
-        let settings = UIUserNotificationSettings(types: .alert, categories: [sentTabsCategory])
-
-        application.registerUserNotificationSettings(settings)
```
Comment (jhugman) [inline Client/Helpers/FxALoginHelper.swift]: YES!

**Pull request merged**